Makefile

```diff
@@ -185,6 +188,8 @@ failpoint-ctl:
 	$(GO) get github.com/pingcap/failpoint/failpoint-ctl@v0.0.0-20200210140405-f8f9fb234798
 goimports:
 	$(GO) get golang.org/x/tools/cmd/goimports@v0.0.0-20200309202150-20ab64c0d93f
+gosec:
+	$(GO) get github.com/securego/gosec/cmd/gosec
```

Comment:
Do we need to specify a version for gosec? Like any other `$(GO) get`

Reply:
Agree